patchwork/steps/CreatePR/CreatePR.py

```diff
@@ -147,17 +149,19 @@ def create_pr(
     base_branch_name: str,
     target_branch_name: str,
     scm_client: ScmPlatformClientProtocol,
+    issue_url: str = None,
```

Comment:
Would this be `issue_url: str | None = None`?

Reply:
yep, good catch.